### PR TITLE
Created audio quote block

### LINKED ISSUE #1108: audio quote block
Editors suggested we add an audio quote block like in this article

https://www.nytimes.com/interactive/2023/05/11/opinion/connie-chung-named-after.html?unlocked_article_code=_ciEExomEiuJjAF5TB-tMVjpvEIfyuiPewMW-ITFh6aHNGlFcQWffazK77_YvlIUoZ7w_d1szZdhxa6oQmfNAlRGTykBLjsjXin-FOnic5mdMhJE65ynL6W97ac3bL8b8Q3sqnbZgteXxJuXUF1PqG4w12F1Oah94J8WDAth7ZTRcKqG0wk0igMzd1kypVrg4t972cwAD-0cZAbsQuJmNsZ7ze2m5D2zzIpMZ3U0ZvbQ83vpXDqhdX3PELwPAVGpFTmNXtzhZFwYbrT_mjgytb-8oMTRDaNNIo84ILfGqxBqBqL3m1h9MHMHu-fCdS3qdwROI8dod6jxmwFKWhQlIvpm6PA48yFZf7VfcJfybNX-ig&amp;smid=url-share

### PR DESCRIPTION
**Pull request by SamuelmdLow**
Copied the quote block but had it play audio when clicked

### CODE REVIEW

File: article/models.py [hunk 1]
```diff
@@ -479,6 +480,15 @@ class ArticlePage(SectionablePage, UbysseyMenuMixin):
                 label = "Pull Quote",
                 template = 'article/stream_blocks/quote.html',
             )),
+            ('audio_quote', blocks.StructBlock(
+                [
+                    ('content',blocks.CharBlock(required=False)),
+                    ('source',blocks.CharBlock(required=False)),
```
Comment (FireBoyAJ24) [inline article/models.py:486]: Can you add a help text to explain what the content and source does?

File: article/templates/article/article_page_magazine_2023.html
Comment (FireBoyAJ24) [inline article/templates/article/article_page_magazine_2023.html:1]: image

The quote is not aligned correctly.

**Review by FireBoyAJ24 — CHANGES_REQUESTED** [review]
1. The audio button does not give the sound when I click on it but send me to the top of the webpage.

2. I looked up online and found that audio was not fully supported by wagtail. Do you want to try out the [wagtailmedia](https://github.com/torchbox/wagtailmedia) so that it can allow us to reduce the complexity in the backend?

File: article/templates/article/article_page_magazine_2023.html
Comment (SamuelmdLow) [reply article/templates/article/article_page_magazine_2023.html:1]: I feel like you didn't run `gulp build`

**Review by FireBoyAJ24 — APPROVED** [review]

**Comment by SamuelmdLow** [comment]
Going to put in visual essay

**Pull request closed without merging**